src/components/layout/Modal.tsx

```diff
@@ -1,7 +1,6 @@
-import { Dialog, Transition } from '@headlessui/react';
+import { Dialog, DialogPanel, DialogTitle, Transition, TransitionChild } from '@headlessui/react';
+import { IconButton, XCircleIcon } from '@hyperlane-xyz/widgets';
 import { Fragment, PropsWithChildren } from 'react';
-import XCircle from '../../images/icons/x-circle.svg';
-import { IconButton } from '../buttons/IconButton';
 
 export function Modal({
```

Comment:
Let's remove this component entirely and use the widgets one as I did here: https://github.com/hyperlane-xyz/hyperlane-explorer/pull/114/files#diff-8d43041520d8da688a2e88756b320ba3a8384e7617a930ae8900009eacef0ecc